applications/luci-app-transmission/root/etc/uci-defaults/40_luci-transmission

```diff
@@ -1,11 +0,0 @@
-#!/bin/sh
```

Comment:
I added service triggers in https://github.com/openwrt/packages/pull/11048, so remove ucitrack here.